xml/admin_ceph_upgrade.xml

```diff
@@ -121,8 +121,36 @@
       <para>
        Run the <command>salt-run upgrade.check</command> command to verify that
        the cluster uses the new bucket type <literal>straw2</literal> and that
-       the &adm; is not a storage node.
+       the &adm; is not a storage node. <literal>straw2</literal>  is the default
+       for any newly created buckets.
       </para>
+      <important>
+        <para>
+          The new <literal>straw2</literal> bucket type fixes several limitations
+          in the original <literal>straw</literal> bucket. The previous
+          <literal>straw</literal> buckets would change some mappings that should
+          have changed when a weight was adjusted. <literal>straw2</literal> achieves
```

Comment:
See earlier comment but it's not so easily reworded here. :-(

Reply:
I'm going to leave this one, sorry.